e2e/cypress/fixtures/saml_users.json

```diff
@@ -0,0 +1,242 @@
+        //     "email": "samluser-15@test.com",
+        //     "firstname": "saml15",
+        //     "lastname": "user",
+        //     "userType": ""
+        // },
+        // "samluser-16": {
+        //     "username": "samluser-16",
+        //     "password": "Password1",
+        //     "email": "samluser-16@test.com",
+        //     "firstname": "saml16",
+        //     "lastname": "user",
+        //     "userType": ""
+        // },
+        // "samluser-17": {
+        //     "username": "samluser-17",
+        //     "password": "Password1",
+        //     "email": "samluser-17@test.com",
+        //     "firstname": "saml17",
+        //     "lastname": "user",
+        //     "userType": ""
+        // },
+        // "samluser-18": {
+        //     "username": "samluser-18",
+        //     "password": "Password1",
+        //     "email": "samluser-18@test.com",
+        //     "firstname": "saml18",
+        //     "lastname": "user",
+        //     "userType": ""
+        // },
+        // "samluser-19": {
+        //     "username": "samluser-19",
+        //     "password": "Password1",
+        //     "email": "samluser-19@test.com",
+        //     "firstname": "saml19",
+        //     "lastname": "user",
+        //     "userType": ""
+        // },
+        // "samluser-20": {
+        //     "username": "samluser-20",
+        //     "password": "Password1",
+        //     "email": "samluser-20@test.com",
+        //     "firstname": "saml20",
+        //     "lastname": "user",
+        //     "userType": ""
+        // },
+        // "samluser-21": {
+        //     "username": "samluser-21",
+        //     "password": "Password1",
+        //     "email": "samluser-21@test.com",
+        //     "firstname": "saml21",
+        //     "lastname": "user",
+        //     "userType": ""
+        // },
+        // "samluser-22": {
+        //     "username": "samluser-22",
+        //     "password": "Password1",
+        //     "email": "samluser-22@test.com",
+        //     "firstname": "saml22",
+        //     "lastname": "user",
+        //     "userType": ""
+        // },
+        // "samluser-23": {
+        //     "username": "samluser-23",
+        //     "password": "Password1",
+        //     "email": "samluser-23@test.com",
+        //     "firstname": "saml23",
+        //     "lastname": "user",
+        //     "userType": ""
+        // },
+        // "samluser-24": {
+        //     "username": "samluser-24",
+        //     "password": "Password1",
+        //     "email": "samluser-24@test.com",
+        //     "firstname": "saml24",
+        //     "lastname": "user",
+        //     "userType": ""
+        // },
+        // "samluser-25": {
+        //     "username": "samluser-25",
+        //     "password": "Password1",
+        //     "email": "samluser-25@test.com",
+        //     "firstname": "saml25",
+        //     "lastname": "user",
+        //     "userType": ""
+        // }
```

Comment:
Can we remove these users rather than commenting them. It will be much cleaner that way. We can always add these users back in future when we need them.

Reply:
Didn't want to lose the hard work of creating them =),  but sure, I can clean them up.